core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -1878,7 +1878,7 @@ TimeoutException coordinatorTimeoutWaitingForView(int expectedViewId, int curren
    void marshallersNotSupported();
 
    @LogMessage(level = WARN)
-   @Message(value = "infinispan-boss-marshalling dependency detected, jboss-marshalling has been deprecated and will be removed in the future", id = 554)
+   @Message(value = "jboss-marshalling has been deprecated and will be removed in a future release", id = 554)
```

Comment:
```suggestion
   @Message(value = "jboss-marshalling is deprecated and planned for removal", id = 554)
```

Reply:
probably being a bit pedantic here and need to update the contributor's guide but we should prefer present tense over future. "planned for removal" is better than "will be removed" because it avoids making a promise to users.

Reply:
No worries, updated